src/SFA.DAS.AssessorService.Application/Handlers/ao/GetStandardsByAssessmentOrganisationHandler.cs

```diff
@@ -10,23 +10,25 @@
 
 namespace SFA.DAS.AssessorService.Application.Handlers.ao
 {
-    public class GetStandardsByAssessmentOrganisationHandler: IRequestHandler<GetStandardsByOrganisationRequest, List<OrganisationStandardSummary>>
+    /// <summary>
+    /// This handler doesn't restrict based on efffective from / to and status of live
+    /// As it's called from the Admin Service and the Admin side needs to know about active and withdrawn standards.
+    /// </summary>
+    public class GetStandardsByAssessmentOrganisationHandler: IRequestHandler<GetAllStandardsByOrganisationRequest, List<OrganisationStandardSummary>>
```

Comment:
Can we update handler name to match with the request and service method?

Reply:
Have renamed 👍